tests_integration/plugins/options/plugin.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+module.exports = {
+  languages: [
+    {
+      name: "foo",
+      parsers: ["foo-parser"],
+      extensions: [".foo"],
+      since: "1.0.0"
+    }
+  ],
+  parsers: {
+    "foo-parser": {
+      parse: text => ({ text }),
+      astFormat: "foo-ast"
+    }
+  },
+  printers: {
+    "foo-ast": {
+      print: (path, options) =>
+        options.fooOption ? `foo:${options.fooOption}` : path.getValue().text,
+      options: {
```

Comment:
I'm not sure if plugins should belong to the printer or just the plugin. There could be options associated with parsers, too.

Reply:
IMO, options should be plugin-level, but that's the current behavior. It seems we're going to change it? then it should be in another PR.

Reply:
👍  Happy for it to be a separate PR